layout/pages/zoning/zoning.xml

```diff
@@ -310,13 +315,15 @@
 				</Panel>
 			</Panel>
 		</Panel>
-		<Panel class="zoning__button-box">
-			<Button class="button button--blue ml-2 h-align-right" onactivate="ZoneMenuHandler.saveZones()">
-				<Label class="button__text" text="#Zoning_Save" />
-			</Button>
-			<Button class="button ml-2 h-align-right" onactivate="ZoneMenuHandler.cancelEdit()">
-				<Label class="button__text" text="#Zoning_Discard" />
-			</Button>
+		<Panel class="zoning__button-box w-full">
+			<Panel class="h-align-right flow-right">
+				<Button class="button button--blue ml-2 h-align-right" onactivate="ZoneMenuHandler.saveZones()">
+					<Label class="button__text" text="#Zoning_Save" />
+				</Button>
+				<Button class="button ml-2 h-align-right" onactivate="ZoneMenuHandler.cancelEdit()">
+					<Label class="button__text" text="#Zoning_Discard" />
+				</Button>
+			</Panel>
```

Comment:
Are these changes just left over from when the button was added to this area at first, or was this meant to fix something unrelated to the new button?

Reply:
these are just left over. Visually no change.

I'll revert, thanks for reviewing!